Starwatch.Core/Monitoring/AnnouncementMonitor.cs

```diff
@@ -40,15 +40,23 @@ public class Announcement
         public bool EnableMonitor { get; set; }
 
         public Announcement[] Announcements = new Announcement[0];
```

Comment:
PascalCase for Properties.
camelCase for fields.

Either make property or field. If field, make it protected, private, or consider making it a property (just OOP design. Not required but keeps stuff consistent....er).